packages/shared/src/components/cards/list/PlaceholderList.tsx

```diff
@@ -24,10 +24,16 @@ export const PlaceholderList = forwardRef(function PlaceholderCard(
         </CardSpace>
       </CardTextContainer>
       <CardSpace className="my-4">
-        <Text className="h-4 w-3/4" />
-        <Text className="mt-2 h-4 w-1/2" />
+        <CardSpace className="flex">
```

Comment:
This is more inline with how the cards actually look:
![Screenshot 2024-06-18 at 16 20 01](https://github.com/dailydotdev/apps/assets/554874/e9a75260-5505-4e9c-a95c-c8f2a3bd552c)